### PR TITLE
addition of documentation for ui-dropdown

### PR DESCRIPTION
**Pull request by randallkanna**
@pzuraq

### CODE REVIEW

File: tests/dummy/app/controllers/application.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import Ember from 'ember';
 
 export default Ember.Controller.extend({
   actions: {
-    doSomething(test) {
```
Comment (randallkanna) [inline tests/dummy/app/controllers/application.js]: argument not being used

File: tests/dummy/app/templates/application.hbs [hunk 2]
```diff
@@ -14,12 +14,10 @@
     </button>
   {{/ui-dropdown}}
 
-
```
Comment (randallkanna) [inline tests/dummy/app/templates/application.hbs]: removed additional spaces here

**Comment by twokul** [comment]
travis is angry

File: README.md [hunk 3]
```diff
@@ -1,6 +1,44 @@
 # Ui-dropdown
 
-This README outlines the details of collaborating on this Ember addon.
+## Usage
+This Ember addon provides a `ui-dropdown` component that can be easily attached to elements on your page.
```
Comment (twokul) [inline README.md]: extra line above

File: README.md [hunk 4]
```diff
@@ -1,6 +1,44 @@
...
+
+To implement the component, select the id of the element you want to attach the dropdown to.
+
+**Example:**
+
+```
```
Comment (twokul) [inline README.md]: syntax (`hbs`)

File: README.md [hunk 5]
```diff
@@ -1,6 +1,44 @@
...
+
+{{#ui-dropdown target="#center_button" align="center"}}
+  center content
+{{/ui-dropdown}}
+```
+
+**Example:**
+
+You can also add a button inside the dropdown to close the dropdown. 
+```
```
Comment (twokul) [inline README.md]: extra line above & syntax (`hbs`)

File: README.md [hunk 6]
```diff
@@ -1,6 +1,44 @@
...
+<button id="left_button" class="btn btn-default">Left Aligned</button>
+
+{{#ui-dropdown target="#left_button" align="left" as |close|}}
+  left content
+  <button {{action close}}>
+    Close Me!
+  </button>
+{{/ui-dropdown}}
+
```
Comment (twokul) [inline README.md]: remove extra line

File: README.md [hunk 7]
```diff
@@ -1,6 +1,44 @@
...
+```
+
+## Styles
+The styles for `ui-dropdown` can be customized by modifying the various dropdown classes such as `.ff-dropdown-content`, `.ff-dropdown-arrow`, and `.ff-dropdown`.
```
Comment (twokul) [inline README.md]: extra line above

Comment (twokul) [inline README.md]: extra line above

File: README.md [hunk 8]
```diff
@@ -1,6 +1,44 @@
...
+
+Ember Tether should also be set to absolute positioning.
+```
```
Comment (twokul) [inline README.md]: extra line above & syntax (`css`)

**Pull request closed without merging**